$MirOS: ports/misc/mc/patches/patch-src_main_h,v 1.1.7.1 2005/03/18 15:50:29 tg Exp $
--- src/main.h.orig	Fri Jul  1 15:46:44 2005
+++ src/main.h	Mon Feb 20 18:32:46 2006
@@ -124,7 +124,7 @@ extern struct WMenu *the_menubar;
 void done_menu (void);
 void init_menu (void);
 
-#define MC_BASE "/.mc/"
+#define MC_BASE "/.etc/mc/"
 
 struct WPanel;
 void directory_history_add   (struct WPanel *panel, const char *dir);
